packages/flame/lib/src/timer.dart

```diff
@@ -1,18 +1,16 @@
 import 'dart:math';
 import 'dart:ui';
 
-import 'components/component.dart';
-
 /// Simple utility class that helps handling time counting and implementing
 /// interval like events.
 class Timer {
   final double limit;
-  VoidCallback? callback;
+  VoidCallback? onTick;
   bool repeat;
   double _current = 0;
   bool _running = false;
```

Comment:
Shouldn't this have the same behaviour as `TimerComponent` and start directly on creation?

Reply:
Like having the `bool autoStart = true,` as an argument?